### PR TITLE
Add English version of Acre nad AF_Medical articles

### PR DESCRIPTION
**Pull request by N4ki**
These changes will:

- title

### CODE REVIEW

File: docs/en/mods/armaforces_medical.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+| Wound\Dressing| Field Dressing |  Elastic  |  Packing | QuikClot |
+|:-----------:|:--------------:|:---------:|:--------:|:--------:|
+|   Abrasion  |       ++       |           |          |          |
+|  Avulsion   |                |     ++    |          |          |
+|  Contusion  |                |     ++    |          |          |
+|    Crush    |                |     ++    |          |          |
+|     Cut     |       ++       |           |          |          |
+|  Laceration |       ++       |           |          |          |
+|   Velocity  |        +       |      +    |    ++    |    ++    |
+|   Puncture  |        +       |      +    |    ++    |    ++    |
```
Comment (3Mydlo3) [inline docs/en/mods/armaforces_medical.md:26]: Slight reformat so it looks better in raw text.
```suggestion
| Wound\Dressing | Field Dressing | Elastic | Packing | QuikClot |
| :------------: | :------------: | :-----: | :-----: | :------: |
|    Abrasion    |       ++       |         |         |          |
|    Avulsion    |                |   ++    |         |          |
|   Contusion    |                |   ++    |         |          |
|     Crush      |                |   ++    |         |          |
|      Cut       |       ++       |         |         |          |
|   Laceration   |       ++       |         |         |          |
|    Velocity    |       +        |    +    |   ++    |    ++    |
|    Puncture    |       +        |    +    |   ++    |    ++    |
```

File: docs/en/mods/armaforces_medical.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- chance of bleeding occurring in aforementioned timeframe (if bleeding doesn't happen under these terms, it won't happen at all),
+- how many wounds of specific size you can dress using specific dressing
+
+| Efficiency            |      | +    | ++    |
+|-----------------------|:----:|:----:|:-----:|
+| Bleeding timeframe    | 5-10 | 6-20 | 10-30 |
+| Bleeding chance       | 0.4  | 0.3  | 0.1   |
+| Large wounds          | 2    | 4    | 8     |
+| Medium wounds         | 3    | 5    | 9     |
+| Minor wounds          | 4    | 6    | 10    |
```
Comment (3Mydlo3) [inline docs/en/mods/armaforces_medical.md:40]:
```suggestion
| Efficiency         |       |   +   |  ++   |
| ------------------ | :---: | :---: | :---: |
| Bleeding timeframe | 5-10  | 6-20  | 10-30 |
| Bleeding chance    |  0.4  |  0.3  |  0.1  |
| Large wounds       |   2   |   4   |   8   |
| Medium wounds      |   3   |   5   |   9   |
| Minor wounds       |   4   |   6   |  10   |
```

File: docs/en/mods/armaforces_medical.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Conscious soldiers can use new pain suppressing medication - APAP pills. These little bad boys with high dose of paracetamol are safe to use - they pose little to no chance of overdosing. The drawback is that they can't be used on unconscious units.
+
+| Feature                            | Apap                   | Morphine                                 |
+|------------------------------------|------------------------|------------------------------------------|
+| Max efficiency time                | 10 min                 | 30 min                                   |
+| Delay before kicking in            | 2 min                  | 0.5 min                                  |
+| Painkilling strength               | 0.6                    | 0.8                                      |
+| Use case                           | Conscious units in pain| Great pain and unconscious units in pain |
+| Side effects                       | None                   | Lowers pulse                             |
+| Chance of inducing unconsciousness | Negligible             | Very high, 2 injections might lead to loss of consciousness|
```
Comment (3Mydlo3) [inline docs/en/mods/armaforces_medical.md:53]:
```suggestion
| Feature                            | Apap                    | Morphine                                                    |
| ---------------------------------- | ----------------------- | ----------------------------------------------------------- |
| Max efficiency time                | 10 min                  | 30 min                                                      |
| Delay before kicking in            | 2 min                   | 0.5 min                                                     |
| Painkilling strength               | 0.6                     | 0.8                                                         |
| Use case                           | Conscious units in pain | Great pain and unconscious units in pain                    |
| Side effects                       | None                    | Lowers pulse                                                |
| Chance of inducing unconsciousness | Negligible              | Very high, 2 injections might lead to loss of consciousness |
```

File: docs/en/mods/armaforces_medical.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+## Tourniquets 
+
+These little devices are tight - literally. They can be put on all four limbs to stop bleeding occurring within correspondent part:
+
+- can be only applied to arms and legs,
+- drastically faster to apply than dressings, 
+- multiple use - after taking them off, they go back to the backpack of person detaching them from limbs,
+- after a few minutes they induce a growing pain on the person wearing them,
+- medical injections (e.g. blood, saline, morphine) are inefficient when injected through limbs with tourniquet applied
```
Comment (3Mydlo3) [inline docs/en/mods/armaforces_medical.md:63]:
```suggestion
- Can be only applied to arms and legs,
- Drastically faster to apply than dressings, 
- Multiple use - after taking them off, they go back to the backpack of person detaching them from limbs,
- After a few minutes they induce a growing pain on the person wearing them,
- Medical injections (e.g. blood, saline, morphine) are inefficient when injected through limbs with tourniquet applied.
```

**Review by 3Mydlo3 — CHANGES_REQUESTED** [review]
Nice, I grinned several times. 🙂

File: docs/en/mods/armaforces_medical.md (hunk 3 above)
Comment (N4ki) [reply docs/en/mods/armaforces_medical.md]: Welp, I should really put my interest towards these markdown VS plugins ;)

**Comment by 3Mydlo3** [comment]
Please also remove English parts from Polish medical article.

File: docs/en/mods/armaforces_medical.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
+﻿# ArmaForces - Medical
+
+## Main changes
+
+- dressings made more efficient
+- medical actions take less time to complete
+- medical system made friendlier to the players (no need to keep bandaging yourself as often as before, also munch on these apap pills with extra paracetamol to easily fight the pain without worry of losing consciousness)
```
[the remark this answers is not in the document]
Comment (N4ki) [reply docs/en/mods/armaforces_medical.md]: I split the last point a little bit, noticed that it basically repeats what first points says. Changes are shorter now and more straightforward (I hope).

**Review by 3Mydlo3 — APPROVED** [review]

**Pull request merged**